plugins/README.md

```diff
@@ -81,30 +81,30 @@ here: https://github.com/proxy-wasm/spec/tree/master
 Service Extension plugins currently support a subset of the ProxyWasm spec.
 Support will grow over time. The current feature set includes:
 
-*   Root context lifecycle callbacks
+*   Root context lifecycle callbacks (host -> wasm)
     *   on_context_create
     *   on_vm_start
     *   on_configure
     *   on_done
     *   on_delete
-*   Stream context lifecycle callbacks
+*   Stream context lifecycle callbacks (host -> wasm)
     *   on_context_create
     *   on_done
     *   on_delete
-*   Stream context HTTP callbacks
+*   Stream context HTTP callbacks (host -> wasm)
     *   on_request_headers
     *   on_response_headers
-*   Stream context HTTP methods
+*   Stream context HTTP methods (wasm -> host)
     *   send_local_response
     *   get__header_map_value, add_header_map_value, replace_header_map_value,
```

Comment:
nit: looks like there's an extra '_' in get_header_map_value

Reply:
Done